rubrics/services/pom.xml

```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.sakaiproject.rubrics</groupId>
+    <artifactId>rubrics-service</artifactId>
+    <version>12-SNAPSHOT</version>
+    <packaging>war</packaging>
+
+    <name>Sakai Rubrics - REST API &amp; Web Components</name>
+    <description>Sakai Native Rubrics Service - Provides a REST API and Web Components for Rubrics</description>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>1.5.1.RELEASE</version>
+        <relativePath/><!-- Disable local parent pom resolution https://issues.apache.org/jira/browse/MNG-5146 -->
+    </parent>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <start-class>org.sakaiproject.rubrics.App</start-class>
+        <java.version>1.8</java.version>
+        <log4jdbc.log4j2.version>1.16</log4jdbc.log4j2.version>
+        <rest.assured.version>2.9.0</rest.assured.version>
+        <spring.security.version>4.2.1.RELEASE</spring.security.version>
+        <spring.boot.version>1.5.2.RELEASE</spring.boot.version>
+        <hibernate.version>4.3.11.Final</hibernate.version>
```

Comment:
Ok I will edit that in both branches directly from github. 

The problem is that SAK-32152 depends on this branch, so any change here should be included in that one before the last commit on that one and I'm trying to maintain that. But if it is ok...to add additional commits in the PR, I'll do in that way. Much more easy for me. Thanks